submissions/vovan-zt/OOP-exerciseTW2/index.js

```diff
@@ -0,0 +1,55 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+*/
+class WorldInhabitant {
+   constructor(species, name, gender, saying) {
+      this.species = species;
+      this.name = name;
+      this.gender = gender;
+      this.saying = saying;
+   }
+      
+    getMeaning() {
+      return `${this.species}; ${ this.name}; ${this.gender}; ${this.saying}; `
+    }
+   
+}
+
+class Human extends WorldInhabitant { 
+   constructor(species, name, gender, saying, legs = 2, hands = 2, friends = '') {
+      super(species, name, gender, saying);
+      this.legs = legs;
+      this.hands = hands;
+      this.friends = friends;
+   }
+
+   getMeaning() {
+      return super.getMeaning() + `${this.legs}; ${this.hands}; ${this.friends}`;
```

Comment:
As a peer developer on the project I am tasked with adding 10 more properties to humans. Please optimize this fragment so that I'd need as small amount of keystrokes to introduce these new properties as only possible. Ideally, I'd like to just list new properties in this method.

Reply:
Хотел это изначально сделать, но не получалось, попробую  исправить.

Reply:
Не думаю что идеально но  поменял немного код.